sys/posix/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menu "Posix"
+    depends on TEST_KCONFIG
+
+config MODULE_POSIX_HEADERS
+    bool
+
+config MODULE_POSIX_SLEEP
+    bool "Posix Sleep"
+    depends on MODULE_ZTIMER_MSEC
+    depends on MODULE_ZTIMER_USEC
+    select MODULE_POSIX_HEADERS
```

Comment:
Would it make sense to have the corresponding symbols closer to the modules? So this would be in `posix/sleep/Kconfig`.

Reply:
Done

Reply:
...wait a sec